### DISCUSSION TITLE
Cannot setup in docker

### DISCUSSION THREAD

**Discussion by AXT2185** [Xilinx/finn#1059] [Q&A] [answered]
Hello, when I want to execute by docker. it will push error like this.
![屏幕截图 2024-04-24 093420](https://github.com/Xilinx/finn/assets/85671658/ff4c02bc-bd73-401d-bdc0-6bcae0169905)
![屏幕截图 2024-04-24 094710](https://github.com/Xilinx/finn/assets/85671658/694e7ecb-cfab-4787-bcc8-544a0ed078b4)
How to solve it? And this is my environment variables configuration. Anything wrong?
![屏幕截图 2024-04-24 093558](https://github.com/Xilinx/finn/assets/85671658/5f4f9348-742a-470f-b86d-0b00900885ad)

**Comment by fpjentzsch** [comment, marked as answer]
Looks like an internet connection problem to github.com

**Reply by AXT2185** [reply]
Yes, it worked after I tried it a few more times
